src/libs/OptionsListUtils.ts

```diff
@@ -2479,8 +2479,10 @@ function filterOptions(options: Options, searchInputValue: string, config?: Filt
     };
 }
 
-function sortItemsAlphabetically<T extends HasText>(membersList: T[]): T[] {
-    return membersList.sort((a, b) => (a.text ?? '').toLowerCase().localeCompare((b.text ?? '').toLowerCase()));
+function sortAlphabetically<T extends HasText>(items: T[]): T[];
+function sortAlphabetically<T extends Record<TKey, string | undefined>, TKey extends keyof T>(items: T[], key: TKey): T[];
+function sortAlphabetically<T extends Record<TKey, string | undefined>, TKey extends keyof T>(items: T[], key: TKey = 'text' as TKey): T[] {
```

Comment:
@blazejkustra NAB: Please make it more readable. Can we make the second parameter mandatory in all cases

Reply:
Sure! https://github.com/Expensify/App/pull/47928/commits/f7a701a8f31c864713fc23808171c94a65ee786f